src/fhir/api.ts

```diff
@@ -33,6 +36,15 @@ export const createBundle = (form: FormValues) => {
   const furtherTesting = furtherTestingAndId(form.result, patient.id);
   const plan = planDefinitionAndId(form.sample, form.result, patient.id);
   const { authoriser, reporter } = practitionersAndIds(form.result);
+  let variants: ResourceAndIds[];
+  if (form.variant.length) {
+    variants = form.variant.map((variant: VariantSchema) =>
+      variantAndId(variant, patient.id, specimen.id, specimen.identifier, reporter.id, authoriser.id),
+    );
+  } else {
+    variants = [createNullVariantAndId(patient.id, specimen.id, specimen.identifier, reporter.id, authoriser.id)];
+  }
```

Comment:
ah interesting, yes when I was testing this out there was no pre-populated data in the form so the array was empty. 

Though I think this should be implemented in #22? because there's the helper component which deals with adding a removing from arrays in a form dynamically - so would be able to deal with this without having to manually implement these parts

Reply:
Ok, let's tackle this in the next PR?